src/app/src/main/kotlin/org/imperial/mrc/hint/AppStart.kt

```diff
@@ -32,6 +32,7 @@ class MvcConfig(val config: Config) : WebMvcConfigurer {
         //Ajax endpoints only available to logged in users
         registry.addInterceptor(SecurityInterceptor(config, ""))
                 .addPathPatterns("/adr/**")
+                .addPathPatterns("/user/**")
```

Comment:
I noticed that the project related endpoints are not locked down to logged in users - is this intentional?